test/fixtures/output/local/kafka/topic/configuration/list.golden

```diff
@@ -0,0 +1,28 @@
+                   Name                   |        Value         
+------------------------------------------+----------------------
+  cleanup.policy                          | delete               
+  compression.type                        | producer             
+  delete.retention.ms                     | 86400000             
+  file.delete.delay.ms                    | 60000                
+  flush.messages                          | 9223372036854775807  
+  flush.ms                                | 9223372036854775807  
+  follower.replication.throttled.replicas |                      
```

Comment:
Why are some of these empty?

Reply:
I think the default value for those is just empty; perhaps there are no throttled replicas.

Reply:
I wonder if we should print `[]` here... ok to ignore for now, but maybe open a ticket if you agree?